packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -451,15 +472,100 @@ export class DbTargetGlobal {
      * until the backend has caught up to this snapshot version again. This
      * prevents our cache from ever going backwards in time.
      */
-    public lastRemoteSnapshotVersion: DbTimestamp
+    public lastRemoteSnapshotVersion: DbTimestamp,
+    /**
+     * A cache of the number of targets persisted.
```

Comment:
Nit: The number of targets persisted.

This doesn't strike me as a cache (especially since we have use the word Cache extensively in this file).

Reply:
Done.